currency_rate_inverted/readme/CONTRIBUTORS.rst

```diff
@@ -3,7 +3,6 @@
 * Statecraft Systems Pte Ltd (www.statecraftsystems.sg)
 * Komit (Jean-Charles Drubay <jc@komit-consulting.com>)
 * `CorporateHub <https://corporatehub.eu/>`__
-
-  * Alexey Pelykh <alexey.pelykh@corphub.eu>
-
+* Alexey Pelykh <alexey.pelykh@corphub.eu
```

Comment:
That change removes nested-list formatting. You may use http://rst.ninjs.org/ to evaluate the file before and after